libraries/utilities/analyze_perf_results.py

```diff
@@ -232,8 +232,14 @@ def analze_perf_results(test_id):
 
     (opts, args) = parser.parse_args(arg_parameters)
 
+    try:
+        cluster_conf = os.environ["CLUSTER_CONFIG"]
```

Comment:
I've seen multiple calls to this try/except block, could be moved to a reusable function called `must_get_cluster_config()`, which raises if the env variable is missing